packages/webpack-cli/lib/utils/logger.js

```diff
@@ -1,3 +1,3 @@
-const { logger } = require('@webpack-cli/logger');
+const { getLogger } = require('webpack/lib/logging/runtime');
```

Comment:
`webpack/lib/logging/runtime` should not be used here. Use `compiler.getInfrastructureLogger("webpack-cli")` instead.

Reply:
Will update

Reply:
@sokra we use logging before run webpack and after, and when we do other actions which don't need webpack (like `init`/`info`/`migrate`/`generate`), so we can't use infrastructure logger